.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,13 @@
-Add the story URL here. Prefer the short link format, e.g. https://app.clubhouse.io/acme/story/1234/
+https://github.com/nimblehq/git-template/issues/??
```

Comment:
Should be this repo link
```suggestion
https://github.com/Nihisil/nimble-certification-elixir/issues/??
```

Reply:
I'm sorry, for some reason I missed this suggestion. It will be fixed in future PRs 🙏